metadata-models/src/main/pegasus/com/linkedin/ermodelrelation/ERModelRelationProperties.pdl

```diff
@@ -0,0 +1,79 @@
+namespace com.linkedin.ermodelrelation
+
+import com.linkedin.common.AuditStamp
+import com.linkedin.common.DatasetUrn
+import com.linkedin.common.CustomProperties
+
+/**
+ * Properties associated with a ERModelRelation
+ */
+@Aspect = {
+  "name": "eRModelRelationProperties"
+}
+record ERModelRelationProperties includes CustomProperties {
+
+  /**
+   * Name of the ERModelRelation
+   */
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  name: string
+
+  /**
+  * First dataset in the ermodelrelation (no directionality)
+  */
+   @Relationship = {
+     "name": "ermodelrelationA",
+     "entityTypes": [ "dataset" ]
+   }
+  @Searchable = {
+    "fieldType": "TEXT_PARTIAL",
+    "enableAutocomplete": true,
+    "boostScore": 10.0
+  }
+  datasetA: DatasetUrn
```

Comment:
let's rename this to `source`

Reply:
also lets make the type Urn (instead of the more strict DatasetUrn). That way we can evolve this to other "Dataset"-like things in the future.